Hints.py

```diff
@@ -49,7 +49,7 @@
     'bingo.json',
     'chaos.json',
     'chaos_no_goal.json',
-    'coop2.json',
+    'coop3.json',
```

Comment:
Could this be renamed to just `coop.json` (and the `name` field adjusted accordingly) so it doesn't have to be changed every season? We made a similar change for MW in #2124.

Reply:
done!